packages/slice-machine/tests/server/services/sliceService.test.ts

```diff
@@ -6,17 +6,19 @@ import {
 } from "../../../server/src/api/services/sliceService";
 import { upload } from "../../../server/src/api/services/uploadScreenshotClient";
 
-import { Client } from "../../../lib/models/server/Client";
-import { ApplicationMode } from "../../../lib/models/server/ApplicationMode";
 import allFieldSliceObject from "../../__mocks__/sliceModel";
 import backendEnvironment from "../../__mocks__/backendEnvironment";
+import { Client, ApplicationMode } from "@slicemachine/client";
 import { Slices } from "@slicemachine/core/build/models/Slice";
 import { resolvePathsToScreenshot } from "@slicemachine/core/build/libraries/screenshot";
 
 const allFieldSliceModel = Slices.toSM(allFieldSliceObject);
 
-jest.mock("../../../lib/models/server/Client", () => {
+jest.mock("@slicemachine/client", () => {
+  const originalModule = jest.requireActual("@slicemachine/client");
```

Comment:
would copying the nock objects from the client also work, at least that way you can be double sure it works as expected?

Reply:
That test was like one the first I fixed 😓.
In the end it's just testing that we call the right functions.

The fact that we make the right call is already checked in the tests of the Client.